src/api/config/routes/api_routes.rb

```diff
@@ -1,4 +1,4 @@
-require_relative 'routes_helper'
+require File.join(Rails.root, 'config', 'routes', 'routes_helper')
```

Comment:
This can be replaced by `require_relative 'routes/routes_helper'` at the top of `config/routes.rb`. Same thing for `webui_routes.rb`.

Reply:
did you try it? `require_relative` and `instance_load` doesn't work well. hint try it locally.

Reply:
Hi @dmarcoux , no unfortunately it cannot be replaced. Because of the `instance_eval`, the basepath for the `require_relative` cannot be inferred. Or did you try it locally and worked for you?